piou-piou/ribs-admin-bundle/0.1/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+    "bundles": {
+        "PiouPiou\\RibsAdminBundle\\RibsAdminBundle": ["all"]
```

Comment:
Isnt you bundle called: `Ribs\\RibsAdminBundle\\RibsAdminBundle`?

Reply:
Yes but it's an error that I have done before. I have installed a recipe server. I test all things on my server and i will push it on this merge later :)

Reply:
all is fixed in last commit, you can close you review :)